### PR TITLE
Add bot configurations

### PR DESCRIPTION
**Pull request by rgreinho**
Adds the configurations for the following bots:
- dependabot
- kodiak
- stale

The bots will have to be enabled by an admin in order to get activated.

Fixes guacsec/guac#55

Signed-off-by: Rémy Greinhofer <remy.greinhofer@gmail.com>

### CODE REVIEW

**Review by mihaimaruseac — APPROVED** [review]
LGTM

**Comment by rgreinho** [comment]
The [PR size]( https://github.com/marketplace/pull-request-size) bot will also need to be activated, but does not have any configuration.

**Comment by rgreinho** [comment]
The [PR size](https://github.com/marketplace/pull-request-size) bot will also need to be activated, but does not have any configuration.

File: .github/stale.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Number of days of inactivity before a PR becomes stale.
+daysUntilStale: 60
```
Comment (lumjjb) [inline .github/stale.yml:2]: i think we agreed on 90 here. other than that, lgtm

**Comment by lumjjb** [comment]
bots enabled!

File: .github/stale.yml (hunk 1 above)
Comment (rgreinho) [reply .github/stale.yml:2]: And we did :) 

If you look at the line bellow, it says "daysUntilClose: 30", so 60 days for a warning + 30 days before closing, and we're at 90 as discussed.

**Review by pxp928 — APPROVED** [review]

**Pull request merged**